src/Angular.js

```diff
@@ -193,7 +193,8 @@ msie = document.documentMode;
 
 function isNodeList(obj) {
   return typeof obj.length == 'number' &&
-         typeof obj.item == 'function';
+         (typeof obj.item == 'function' ||
+          isString(obj.jquery));
```

Comment:
It is actually easier to identify jQuery and jqLite instances using `instanceof jqLite`.